libs/remix-ui/run-tab/src/lib/components/recorderCardUI.tsx

```diff
@@ -22,24 +25,45 @@ export function RecorderUI (props: RecorderProps) {
         <div className="d-flex">
           <label className="mt-1 udapp_recorderSectionLabel">Transactions recorded</label>
           <div className="ml-2 mb-2 badge badge-pill badge-primary" title="The number of recorded transactions">{props.count}</div>
+        <OverlayTrigger placement={'right'} overlay={
+          <Tooltip className="text-nowrap" id="info-recorder">
+            <span>Save transactions (deployed contracts and function executions) and replay then in an
+            another environment. <br/> e.g Transactions created in Javascript VM can be replayed in the Injected Web3.
+          </span>
+          </Tooltip>
+        }>
+          <i style={{ fontSize: 'medium' }} className={'ml-2 fal fa-info-circle'} aria-hidden="true"></i>
+        </OverlayTrigger>
         </div>
         <div>
           <span data-id='udappRecorderTitleExpander' onClick={toggleClass}>
             <i className={!toggleExpander ? 'fas fa-angle-right' : 'fas fa-angle-down'} aria-hidden="true"></i>
           </span>
         </div>
       </div>
-      <div className={`border-bottom flex-column ${toggleExpander ? "d-flex" : "d-none"}`}>
-        <div className="p-2 mt-2">
-          All transactions (deployed contracts and function executions) can be saved and replayed in
-          another environment. e.g Transactions created in Javascript VM can be replayed in the Injected Web3.
-        </div>
-        <div className="mb-2 udapp_transactionActions">
-          <i className="fas fa-save savetransaction udapp_recorder udapp_icon"
-            onClick={triggerRecordButton} title="Save Transactions" aria-hidden="true">
-          </i>
-          <i className="fas fa-play runtransaction udapp_runTxs udapp_icon" title="Run Transactions" data-id="runtransaction" aria-hidden="true" onClick={handleClickRunButton}></i>
-        </div>
+      <div className={`flex-column ${toggleExpander ? "d-flex" : "d-none"}`}>
+        <div className="mb-1 mt-1 fmt-2 custom-control custom-checkbox mb-1">
+          <input ref={inputLive} type="checkbox" id="livemode-recorder" data-id="runtabLivemodeInput" className="custom-control-input custom-select" name="input-livemode"/>
+          <label className="form-check-label custom-control-label" htmlFor="livemode-recorder">When running transactions, get the contracts bytecodes from the last compilation result.</label>
+        </div>     
+        <div className="mb-1 mt-1 udapp_transactionActions">
+        <OverlayTrigger placement={'right'} overlay={
+          <Tooltip className="text-nowrap" id="tooltip-save-recorder">
+            <span>Save {props.count} transaction(s) to JSON.
```

Comment:
Either `to JSON` can be removed to make it `Save {props.count} transaction(s)` or
it can be `Save {props.count} transaction(s) as scenario file` to keep it similar to Run button tooltip